### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by AndreyStrogush**
second branch

### CODE REVIEW

File: src/carExample/Car.java [hunk 1]
```diff
@@ -1,172 +1,172 @@
...
-•	Нет пустого конструктора. Так как есть поля в классе, которые нельзя изменять после создания объекта. Например дата производства.
-•	Конструктор с датой производства.
-•	Конструктор со всеми полями, кроме массива колес и массива дверей.
+package carExample;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class Car {
+    private final String productionDate;
```
Comment (Taniann) [inline src/carExample/Car.java:7]: use Date or LocalDate instead of String for date

File: src/carExample/Car.java [hunk 2]
```diff
@@ -1,172 +1,172 @@
...
+
+    //set the default set of doors
+    public void setTheDefoltSetOfDoors() {
+        for (int door =0; door < 5; door++) {
+            doors.add(new CarDoors(false, false));
+        }
+
+    }
+
+    public void printInfoAboutCar() {
```
Comment (Taniann) [inline src/carExample/Car.java:119]: better override toString for that purpose

File: src/carExample/CarDoors.java [hunk 3]
```diff
@@ -1,79 +1,79 @@
...
-•	закрыть дверь
-•	открыть/закрыть дверь (если дверь открыта и вызывается эта функция, значит дверь необходимо закрыть и наоборот)
-•	открыть окно
-•	закрыть окно
-•	открыть/закрыть окно(если дверь открыта и вызывается эта функция, значит дверь необходимо закрыть и наоборот)
-•	Вывести в консоль данные об объекте
+package carExample;
+
+public class CarDoors {
+    private boolean doorIsOpen;
```
Comment (Taniann) [inline src/carExample/CarDoors.java:4]: better isDoorOpened, begin with "is" for boolean variable names

File: src/carExample/MainTest.java [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
-}
+package carExample;
+
+public class MainTest {
+    public static void main(String[] args) {
+        Car myCar = new Car("2015", "Diesel", 180, 6.2d, 7, 2, 0);
+        myCar.setTheDefoltSetOfDoors();
+        myCar.setTheDefoltSetOfWheels();
+
+        while (true) {
```
Comment (Taniann) [inline src/carExample/MainTest.java:9]: where is condition to break loop?

**Review by Taniann — CHANGES_REQUESTED** [review]
read comments

File: src/carExample/Car.java (hunk 2 above)
Comment (AndreyStrogush) [reply src/carExample/Car.java:119]: I made both options

Comment (Taniann) [reply src/carExample/Car.java:119]: > I made both options

ok, but in class Car I didnt see overriding method toString()

Comment (AndreyStrogush) [reply src/carExample/Car.java:119]: yes indeed in this class forgot to override, corrected now sending

**Pull request closed without merging**